extensions/metrics-cluster-feature/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "lens-metrics-cluster-feature",
-  "version": "0.0.1",
+  "version": "6.1.0",
```

Comment:
The reason that was to make sure that any new updates to the extensions ensure that they are always up to date (because newer versions will be overridden during update). But if there are no changes then I don't see the point of ensuring that.

If we make a change to the extensions then we should bump the versions.